fhir_client.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'nokogiri'
   spec.add_dependency 'oauth2', '~> 1.1'
   spec.add_dependency 'rack', '>= 1.5'
-  spec.add_dependency 'rest-client', '~> 2.0'
+  spec.add_dependency 'rest-client', '~> 1.8'
```

Comment:
This should be `>= 1.8` or simply remove the version requirement completely.  Using the `~> 1.8` restriction means rest-client should be greater than 1.8, but less than 2.0 which is going to cause a lot of problems for other users.